lib/craftcms-sass/_mixins.scss

```diff
@@ -419,8 +419,8 @@ $elementBorderRadius: 2px;
 }
 
 @mixin readable {
-  font-size: 16px;
-  line-height: 22px;
+  font-size: 14px;
+  line-height: 20px;
```

Comment:
This file is pulled in from  https://github.com/craftcms/sass , so there can’t be any changes to it directly. If you want to change the font size of Redactor inputs, you’d need to do that from RedactorInput.scss.